.gitlab/custom-jobs-and-variables.yml

```diff
@@ -101,8 +74,10 @@ variables:
 .custom_job:
   artifacts:
     reports:
-      junit: junit.xml
 
+# Note: this part is not used by the 'ruby-baseline' pipeline.
+# FIXME: BUILD_ROOT, TPLS_REPO, TESTS_REPO are not needed here.
+#        Also, the definition of SHARED_REPOS_DIR is wrong.
```

Comment:
Another FIXME: if we want to get the reproducer commands in the log to work, we'll probably need to address this and see if the commands actually work. I'm fine if we don't fix this right now.